lib/testing/src/lib/core/actions/identity/query.service.ts

```diff
@@ -168,6 +169,46 @@ export class QueryService {
         return ApiUtil.waitForApi(apiCall, predicate);
     }
 
+    async getTask(taskName, processInstanceId, appName, status): Promise<any> {
```

Comment:
missing property types, also the `retryCount` should be stored within the `getTask` and passed recursively
```suggestion
    async getTask(taskName: string, processInstanceId: string, appName: string, status: string, retryCount = 15): Promise<any> {
```

Reply:
Modified.